extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -60,10 +61,16 @@ export class AmpScrollableCarousel extends AMP.BaseElement {
   setupBehavior_() {
     this.cancelTouchEvents_();
 
-    this.container_.addEventListener('scroll', this.scrollHandler_.bind(this));
-    this.container_.addEventListener(
+    addSerializedEventListener(this.container_, 'scroll', () =>
```

Comment:
self-review: have to give these functions names for the test to be more meaningful